src/platform/Module.c

```diff
@@ -0,0 +1,373 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <Module.h>
+#include <Log.h>
+
+#include <dlfcn.h>
+#include <string.h>
+#include <stdlib.h>
+
+#include <parson.h>
+
```

Comment:
Don't add headers like this. Put back PlatformCommon.h and combine all headers in there.